src/components/dac_dataset_table/DACDatasetConstants.tsx

```diff
@@ -0,0 +1,51 @@
+import {Styles} from '../../libs/theme';
+
+export const styles = {
```

Comment:
Another approach to this is to use modular css. We have several examples in the codebase for this. If the style objects are being manipulated by components, then I think that approach wouldn't work.